vscode-smart-contracts/scripts/linux-vscode-smart-contracts.Jenkinsfile

```diff
@@ -0,0 +1,72 @@
+pipeline {
+    agent { label 'jenkins-worker' }
+    environment {
+        VERSION = sh(
+            returnStdout: true,
+            script: '''\
+                # Extract version number if not set as parameter
+                [ -z "$VERSION" ] && VERSION=$(node -p "require('vscode-smart-contracts/package.json').version")
+                echo -n "$VERSION"
+            '''.stripIndent()
+        )
+        CARGO_CONCORDIUM_VERSION = "${VERSION}"
+        OUTFILE = "s3://distribution.concordium.software/tools/linux/vscode-smart-contracts_${VERSION}"
+    }
+    stages {
+        stage('precheck') {
+            steps {
+                sh '''\
+                    # Fail if file already exists
+                    totalFoundObjects=$(aws s3 ls "$OUTFILE" --summarize | grep "Total Objects: " | sed "s/[^0-9]*//g")
+                    if [ "$totalFoundObjects" -ne "0" ]; then
+                        echo "$OUTFILE already exists"
+                        false
+                    fi
+                '''.stripIndent()
+            }
+        }
+        stage('build') {
+            agent {
+                docker {
+                    reuseNode true
+                    image 'concordium/base:latest'
+                    args '-u root'
+                }
```

Comment:
Why are we doing this? I don't think we should use the concordium/base image. That is an odd image. We are building a pure node thing, are we not? Isn't there an official image for that?

Reply:
I use `node:16` image instead